app/rooms/commands/preparation-commands.ts

```diff
@@ -549,6 +540,7 @@ export class OnToggleReadyCommand extends Command<
   execute({ client, ready }) {
     try {
       // cannot toggle ready in quick play / ranked / tournament game mode
+      if (this.room.state.gameMode !== GameMode.CUSTOM_LOBBY && ready !== true && ready !== undefined)
```

Comment:
why these 2 consecutive ifs ?

Reply:
the ready button sends `undefined` which is why i added the check.

Reply:
I suppose it can be removed.

Reply:
Wait oh. That's an accidental line addition.